cognite_toolkit/_cdf_tk/commands/_migrate/default_mappings.py

```diff
@@ -4,20 +4,42 @@
 
 from cognite_toolkit._cdf_tk.client.data_classes.migration import ResourceViewMappingApply
 
-_ASSET_ID = "cdf_asset_mapping"
-_EVENT_ID = "cdf_event_mapping"
-_TIME_SERIES_ID = "cdf_time_series_mapping"
-_FILE_METADATA_ID = "cdf_file_metadata_mapping"
+ASSET_ID = "cdf_asset_mapping"
+EVENT_ID = "cdf_event_mapping"
+TIME_SERIES_ID = "cdf_time_series_mapping"
+TIME_SERIES_EXTRACTOR_ID = "cdf_time_series_extractor_mapping"
+FILE_METADATA_ID = "cdf_file_metadata_mapping"
+FILE_METADATA_EXTRACTOR_ID = "cdf_file_metadata_extractor_mapping"
 FILE_ANNOTATIONS_ID = "cdf_file_annotations_mapping"
 ASSET_ANNOTATIONS_ID = "cdf_asset_annotations_mapping"
 
 
 @lru_cache(maxsize=1)
 def create_default_mappings() -> list[ResourceViewMappingApply]:
     """Return the default mappings for migration."""
+    ts_property_mapping = {
+        "name": "name",
+        "description": "description",
+        "isStep": "isStep",
+        "isString": "type",
```

Comment:
How are you doing this mapping, btw? isString is a bool, and type is an enum 🤔

Reply:
Special case: https://github.com/cognitedata/toolkit/blob/main/cognite_toolkit/_cdf_tk/utils/dtype_conversion.py#L238